packages/react-native-renderer/src/ReactNativeRenderer.js

```diff
@@ -128,7 +129,7 @@ function render(
   element: Element<ElementType>,
```

Comment:
Can't comment outside this window, but remove the type from line 109-125?

Reply:
Fixed